internal/config/config.go

```diff
@@ -525,14 +456,14 @@ func parseBGPAdvertisements(ads []bgpAdvertisement, cidrsPerAddresses map[string
 			Communities:         map[uint32]bool{},
 		}
 
-		if rawAd.AggregationLength != nil {
-			ad.AggregationLength = *rawAd.AggregationLength
+		if crdAd.AggregationLength != nil {
+			ad.AggregationLength = int(*crdAd.AggregationLength) // TODO CRD cast
```

Comment:
what does it mean CRD cast?

Reply:
it's another reminder that this could be done in a more elegant way, but I'll remove it